test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpRazorCommentsTest.cs

```diff
@@ -173,5 +176,131 @@ public void UnterminatedRazorCommentInVerbatimBlock()
+                               .Accepts(AcceptedCharacters.None)),
+                    Factory.Markup(Environment.NewLine).With(SpanChunkGenerator.Null),
+                    new MarkupTagBlock(
+                        Factory.Markup("</p>"))
+                    ));
+        }
+
+        [Fact]
+        public void MultipleRazorCommentsInSameLineInMarkup()
+        {
+            ParseDocumentTest(
+                "<p>" + Environment.NewLine
+                + "@**@  @**@" + Environment.NewLine
+                + "</p>",
+                new MarkupBlock(
+                    new MarkupTagBlock(
+                        Factory.Markup("<p>")),
+                    Factory.Markup(Environment.NewLine),
+                    new CommentBlock(
+                        Factory.MarkupTransition(HtmlSymbolType.RazorCommentTransition)
+                               .Accepts(AcceptedCharacters.None),
+                        Factory.MetaMarkup("*", HtmlSymbolType.RazorCommentStar)
+                               .Accepts(AcceptedCharacters.None),
+                        Factory.Span(SpanKind.Comment, new HtmlSymbol(
+                            Factory.LocationTracker.CurrentLocation,
+                            string.Empty,
+                            HtmlSymbolType.Unknown))
+                               .Accepts(AcceptedCharacters.Any),
+                        Factory.MetaMarkup("*", HtmlSymbolType.RazorCommentStar)
+                               .Accepts(AcceptedCharacters.None),
+                        Factory.MarkupTransition(HtmlSymbolType.RazorCommentTransition)
+                               .Accepts(AcceptedCharacters.None)),
+                    Factory.EmptyHtml(),
+                    Factory.Markup("  ").With(SpanChunkGenerator.Null),
+                    new CommentBlock(
+                        Factory.MarkupTransition(HtmlSymbolType.RazorCommentTransition)
+                               .Accepts(AcceptedCharacters.None),
+                        Factory.MetaMarkup("*", HtmlSymbolType.RazorCommentStar)
+                               .Accepts(AcceptedCharacters.None),
+                        Factory.Span(SpanKind.Comment, new HtmlSymbol(
+                            Factory.LocationTracker.CurrentLocation,
+                            string.Empty,
+                            HtmlSymbolType.Unknown))
+                               .Accepts(AcceptedCharacters.Any),
+                        Factory.MetaMarkup("*", HtmlSymbolType.RazorCommentStar)
+                               .Accepts(AcceptedCharacters.None),
+                        Factory.MarkupTransition(HtmlSymbolType.RazorCommentTransition)
+                               .Accepts(AcceptedCharacters.None)),
+                    Factory.Markup(Environment.NewLine).With(SpanChunkGenerator.Null),
+                    new MarkupTagBlock(
+                        Factory.Markup("</p>"))
+                    ));
+        }
```

Comment:
```
<p>
@* abcd *@

</p>
```
```
<p>

@* abcd *@
</p>
```

```
<p>
@* abcd *@ Extra @* efgh *@
</p>
```

```
<p>
@* 
abcd 
*@
</p>
```

Intermingle some content in the above Razor comments just for variation instead of always doing `@**@`.